integration-cli/docker_cli_run_test.go

```diff
@@ -4284,6 +4284,17 @@ func (s *DockerSuite) TestRunCredentialSpecWellFormed(c *check.C) {
 func (s *DockerSuite) TestRunServicingContainer(c *check.C) {
 	testRequires(c, DaemonIsWindows, SameHostDaemon)
 
+	// This functionality does not exist in post-RS3 builds. Note we get the version number
+	// this way as test binaries are not manifested.
+	if runtime.GOOS == "windows" {
+		v, err := kernel.GetKernelVersion()
+		c.Assert(err, checker.IsNil)
+		build, _ := strconv.Atoi(strings.Split(strings.SplitN(v.String(), " ", 3)[2][1:], ".")[0])
```

Comment:
I guess you can't use the Windows `VersionInfo` type, but can you get the build from the second component of the string instead of parsing the third? It looks like the second is the build number:
```go
build, _ := strconv.Atoi(strings.SplitN(v.String(), " ", 3)[1])
```
```go
func (k *VersionInfo) String() string {
	return fmt.Sprintf("%d.%d %d (%s)", k.major, k.minor, k.build, k.kvi)
}
```

Reply:
No....

```
	fmt.Println(v.String())
	build, _ := strconv.Atoi(strings.Split(strings.SplitN(v.String(), " ", 3)[2][1:], ".")[0])
	fmt.Println(build)
	build, _ = strconv.Atoi(strings.SplitN(v.String(), " ", 3)[1])
```

Output (of a non-manifest app, such as the test binaries) on RS3

```
6.2 9200 (16299.15.amd64fre.rs3_release.170928-1534)
16299
9200
```

Reply:
Ah, makes sense. I misunderstood the comment.

It would be nice if the test binaries were built with a manifest, to ensure they receive accurate version information (and perhaps other behavior changes), but that is beyond the scope here.

Reply:
TBH, I'm not sure how to manifest test binaries without some form of wrapper around `go test`, something similar to how the autogen is done for building the daemon. Something I can look at, but not needed for this PR or high priority.